packages/pangea-sdk/pangea/services/audit/audit.py

```diff
@@ -727,6 +727,46 @@ def results(
             self.update_published_roots(response.result)
         return self.handle_results_response(response, verify_consistency, verify_events)
 
+    def log_stream(self, data: dict) -> PangeaResponse[PangeaResponseResult]:
```

Comment:
Just `data: dict`? Don't we want to check that it's an array with an id and each data log? I mean, just `dict` seems to be so generic

Reply:
It might not be like that for all providers.